src/MainWindow.cpp

```diff
@@ -1430,6 +1416,11 @@ void MainWindow::on_editPasteFeatureAction_triggered()
     view()->invalidate(true, true, false);
 }
 
+void MainWindow::dieClipboardInvalid()
+{
+    QMessageBox::critical(this, tr("Clipboard invalid"), tr("Clipboard do not contain valid data."));
```

Comment:
TODO: It should read "Clipboard *does* not contain valid data.". Kept this way for now to avoid updating translations.